testing/cockroach/ldflags_fix.patch

```diff
@@ -0,0 +1,18 @@
+--- a/Makefile
++++ b/Makefile
+@@ -78,12 +78,12 @@
+ build: install
+ 
+ .PHONY: install
+-install: LDFLAGS += $(shell GOPATH=${GOPATH} build/ldflags.sh)
++install: GOLDFLAGS := $(shell GOPATH=${GOPATH} build/ldflags.sh)
```

Comment:
LDFLAGS has implications for gcc and pals.  GOLDFLAGS is something I picked that isn't used by anything else.